### DISCUSSION TITLE
Modulatorvalue issue.

### DISCUSSION THREAD

**Discussion by Ted-juh** [RomanKubiak/ctrlr#150] [Q&A]
As simple as it gets but it's giving me headaches. Sliders seem to be broken for me. As in they do not show the current modulator value in the general properties and thus breaking code depending on it. Combobox and Listbox seem to work fine.  It's not only with new panels, old panels seem to work fine until I spin some modulators for a while or add a new slider. The slider itself is working correct. It shows the right value. When I open a second panel it "does" show the current modulator value in the properties but it remains 0 no matter which way I drag the modulator, shout at it, or show indecent pics of my gf as encouragement.

The most simple panel existing in the Ctrlr Universe but not working for me. Featuring just one slider with no scripts attached:
[modulatorValue.zip](https://github.com/RomanKubiak/ctrlr/files/5917172/modulatorValue.zip)

What I tried so far:
With or without code attached to the modulatorValueChange.
Uninstalling/ re-installing Ctrlr.  (Multiple times, thoroughly)
Remove everything in the "appData/Roaming" directory.
Try the x86/ x64 version.
Spin the modulator for frigging hours until the mouse went dead.

Fyi:
Windows 10 Enterprise
Windows 10 v. 20H2
OS Build: 19042.746

Ctrlr:
Version = 5.3.201, 
Build date = Wed Mar 30 00:52:44 CEDT 2016, 
Branch = Nightly, Juce = 4.0.2, 
libusb = 1.0.19, 
liblo = 0.28,

**Comment by dnaldoog** [comment]
Where is "**general properties**"?

**Reply by Ted-juh** [reply]
I'm talking about "current modulator value" in the properties tab when you open panel mode.

**Reply by dnaldoog** [reply]
I see 59 in that field as it should be!
![modvalue](https://user-images.githubusercontent.com/76831683/106884434-a8002f00-6724-11eb-841b-493b238bb329.png)

**Reply by Ted-juh** [reply]
That's the way it is supposed to be. But it does stay at zero at my end if it shows the current modulator value at all.
